### PR TITLE
Fixes #2 "Bad Request"

### LINKED ISSUE #2: "Bad Request" when Uploading to 3D45
As originally reported by @timmehtimmeh in #1 , there is an issue when selecting the "Start Print Job" checkbox when uploading.

Printer: 3D45
Firmware: June 2021

After slicing, and you click Upload to 3D45, you are prompted for a filename and to check a box for "Filename" and a checkbox for Start print job.

image

If you DO NOT check the box, the file is uploaded to the 3D45 and **it starts to print**.

If you DO check the box, the file is uploaded, and immediately an error message is shown:

Screenshot from 2021-07-11 21-19-36

And it does not start printing.

### PR DESCRIPTION
**Pull request by lukasmwerner**
This patch removes some old code from the where this plugin was based off of.

### CODE REVIEW

**Comment by lukasmwerner** [comment]
The most recent commit changes the workflow where it will upload immediately instead of prompting the user for filename and if they wanted to print. 

TL;DR: Clicking Start print on {printername} will start printing without any further prompts
